### DISCUSSION TITLE
macOS Big Sur 安装失败

### DISCUSSION THREAD

**Discussion by ghost** [daliansky/Hackintosh#274] [Q&A]
ThinkBook 15 IIL 2019 使用黑果小兵macOS Big Sur 11.1 20C69双EFI镜像，MD5已校验。自带Clover引导完全不能启动，已设置UEFI启动，u盘引导仍然启动到windows boot manager。安全启动已关闭，VT-d没有选项。。。使用小新14 IML EFI OC版，已加-v仍没有啰嗦模式。只有一个苹果标志，没有进度条，u盘指示灯闪1分钟后完全灭掉。Clover啰嗦模式卡[EB|#LOG:EXITBS:START] 71...... cfglock好像已经解了。
求助，怎么办？？？谢谢。
配置：
cpu i5-1035G1（ice lake）
gpu 核显UHD graphics 独显（不指望驱动）amd radeon 630
无线网卡，蓝牙 AC9560
有线网卡 只说是realthk gbe

**Comment by Z-fly** [comment]
Radeon 630 不能驱动吧
